lib/resources/r4/types.yaml

```diff
@@ -0,0 +1,6 @@
+CodeableConcept: http://hl7.org/fhir/R4/datatypes.html#CodeableConcept
+dateTime: http://hl7.org/fhir/R4/datatypes.html#dateTime
+Encounter: http://hl7.org/fhir/R4/encounter.html
+id: http://hl7.org/fhir/R4/datatypes.html#id
+Period: http://hl7.org/fhir/R4/datatypes.html#Period
+string: http://hl7.org/fhir/R4/datatypes.html#string
```

Comment:
Update these to use the `https` links.

Reply:
https://github.com/cerner/fhir.cerner.com/pull/240/commits/95e5482ad098fdd0ddd84bb68aded37c3a9b6650#diff-b3c25dd9e50fb8a137ff641afe13c3b7R1